Packs/PcapAnalysis/Scripts/PcapFileExtractor/PcapFileExtractor.py

```diff
@@ -0,0 +1,195 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+import os
+from os.path import isfile
+from tempfile import mkdtemp
+import shutil
+import glob
+import subprocess
+import hashlib
+
+
+def get_pcap_path(args):
+    """
+    :param args: arg from demisto
+    :return: path of pcap file
+    """
+    file_entry_id = ''
+    if 'entry_id' in args:
+        file_entry_id = args.get('entry_id')  # type: ignore
+
+    if not file_entry_id:
+        return_error('You must set entry_id')
+
+    res = demisto.executeCommand('getFilePath', {'id': file_entry_id})
+    if res[0]['Type'] == entryTypes['error']:
+        return_error(f'Failed to get the file path for entry: {file_entry_id}')
+
+    return res[0]['Contents']['path']
+
+
+def find_files_protocol(file_path):
```

Comment:
i dont think you need that function, you can just run `--export-objects` on all protcols

Reply:
I can't, it doesn't work for all protocols in the command.
You must specify one protocol and it should be the right as the type of the file.